README.md

```diff
@@ -1,2 +1,6 @@
 # Assignment_JS
+<<<<<<< HEAD
+Backend Roadmap Javascript assignment
```

Comment:
This file has been submitted without resolving the conflict. And readme and considered "Head" which indicates the conflict at the start as a heading.